src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php

```diff
@@ -25,61 +25,33 @@ public function __construct(array $options = [])
         $this->options = $options;
     }
 
-    public function getDiffusionName(): ?int
-    {
-        return $this->options['diffusion_name'] ?? null;
-    }
-
-    public function getCategory(): ?string
-    {
-        return $this->options['category'] ?? null;
-    }
-
-    public function getFrom(): ?string
-    {
-        return $this->options['from'] ?? null;
-    }
-
     public function getRecipientId(): ?string
     {
-        return $this->options['recipient_id'] ?? null;
+        return null;
     }
 
-    public function setDiffusionName(int $diffusionName): self
+    /**
+     * @return $this
+     */
+    public function diffusionName(int $diffusionName): static
```

Comment:
it's a bit weird to me that an option named "diffusionName" only accepts integers
can you please confirm this is correct @gnito-org?